src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -53,17 +52,19 @@ public static void SetCurrentCircuitHost(CircuitHost circuitHost)
             IClientProxy client,
             RendererRegistry rendererRegistry,
             RemoteRenderer renderer,
-            Action<IComponentsApplicationBuilder> configure,
+            IList<ComponentDescriptor> descriptors,
+            IDispatcher dispatcher,
             IJSRuntime jsRuntime,
             CircuitHandler[] circuitHandlers)
         {
             _scope = scope ?? throw new ArgumentNullException(nameof(scope));
-            Client = client ?? throw new ArgumentNullException(nameof(client));
+            _dispatcher = dispatcher;
+            Client = client;
```

Comment:
Do you now allow client to be null?

Reply:
Yes, because when we prerrender there is no ClientProxy and the plan is that we prerrender using the circuit and then reconnect to the server and attach the client to the circuit.